docs/Configuration.md

```diff
@@ -204,9 +204,10 @@ language from being allowed. If there are more than one `locale tag`
 (with different country codes) for the same language, then
 all those must be removed to block that language.
 
-English is the Zonemaster default language, but it can be blocked
-from being allowed by RPC-API by not including it in the
-configuration.
+The first language in the list will be used as the default for the RPC API.
+If the translation is unavailable it will then default to English.
+Some other part of Zonemaster will default to English directly but it can be
+blocked from being allowed by RPC-API by not including it in the configuration.
```

Comment:
This written too general. The default only applies to the error messages for the API methods that have support for translated error messages.

"If the translation is unavailable it will then default to English." ==> "If translation is not avaiable, then the error messages will be sent untranslated, i.e. in English."

I do not understand:
"Some other part of Zonemaster will default to English directly but it can be
blocked from being allowed by RPC-API by not including it in the configuration."

I think that most of the information should be in API.md. In Configuration.md it only needs to be stated that the first `locale tag` will give the default language for error messages in some API methods that can be seen in the API.md.

Reply:
This section has been updated.